lib/ansible/modules/cloud/vmware/vmware_vsan_health_info.py

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2019, OVH SAS
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: vmware_vsan_health_info
+
+short_description: Gather information about a VMware vSAN cluster's health
+
+version_added: "2.9"
+
+description:
+    - "Gather information about a VMware vSAN cluster's health"
+
```

Comment:
You should mention the extra dependency in the `requirements` section:

https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#documentation-block

Reply:
Done